packages/formatter-json/README.md

```diff
@@ -0,0 +1,18 @@
+# Formatter json (`@sonarwhal/formatter-json`)
+
+The `json` does a `JSON.stringify()` of the results. Output is not very user
+friendly:
```

Comment:
Maybe we should add, something in the lines of:

> friendly, but it can be useful when using it as input for other tools.

To make it clear that even thought it may not be useful for users it is useful in other certain situations.